src/ontology/mondo-edit.obo

```diff
@@ -617799,6 +617814,83 @@ intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.o
+is_a: MONDO:0019118 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391"} ! inherited retinal dystrophy
+intersection_of: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/14550 ! CDHR1
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
+
+[Term]
+id: MONDO:1040086
+name: GUCA1A-related retinopathy
+def: "Any retinopathy caused by a variant in the GUCA1A gene, including cases diagnosed as cone dystrophy 3 or cone-rod dystrophy 14." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
+synonym: "GUCA1A-related retinopathy" EXACT CLINGEN_LABEL [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
+is_a: MONDO:0019118 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391"} ! inherited retinal dystrophy
+intersection_of: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4678 ! GUCA1A
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
+
+[Term]
+id: MONDO:1040087
+name: PRPF3-related retinopathy
+def: "Any retinopathy caused by a variant in the PRPF3 gene." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
+synonym: "PRPF3-related retinopathy" EXACT CLINGEN_LABEL [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391]
+is_a: MONDO:0019118 {source="https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/", source="https://orcid.org/0000-0001-5935-2391"} ! inherited retinal dystrophy
+intersection_of: MONDO:0019118 ! inherited retinal dystrophy
+intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/17348 ! PRPF3
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/8986" xsd:anyURI
+
+[Term]
+id: MONDO:1040088
+name: NRL-related dominant retinopathy
+def: "Any retinopathy caused by a monoallelic variant in the NRL gene. Please note that the mechanism may be increased transactivation." [https://clinicalgenome.org/working-groups/clinical-domain/ocular-cdwg/, https://orcid.org/0000-0001-5935-2391, https://orcid.org/0000-0002-0587-4693]
```

Comment:
"Please note" is not appropriate in a definition. 
Also, what does "the mechanism may be increased transactivation" mean? Mechanism of what? transactivation of what?
Please update.